src/app/common/interceptors/getUser.js

```diff
@@ -0,0 +1,31 @@
+(function () {
+  'use strict';
+  angular
+  .module('healthGuide')
+    .factory('getUserInterceptor', getUserInterceptor);
```

Comment:
I didn't get it.
If I "register the interceptor via an anonymous factory" I can not have it in separate file and insert it into config file like 
`$httpProvider.interceptors.push('getUserInterceptor');
`


> interceptor should be in separate file, consider common/interceprot/authentication.js